squad_albert/metrics.py

```diff
@@ -1,7 +1,5 @@
-from typing import List, Dict
 import tensorflow as tf
-import numpy as np
-from code_loader.inner_leap_binder.leapbinder_decorators import tensorleap_custom_metric
```

Comment:
If possible, let's make minimal changes so that there would be no surprises. Especially if we don't push it back to the platform. Can we make sure imports stay the same?